kedro-datasets/kedro_datasets/spark/spark_streaming_dataset.py

```diff
@@ -20,15 +20,16 @@ class SparkStreamingDataSet(AbstractDataSet):
     `YAML API <https://kedro.readthedocs.io/en/stable/data/\
     data_catalog.html#use-the-data-catalog-with-the-yaml-api>`_:
     .. code-block:: yaml
+
```

Comment:
Looks good, did you test it locally? Maybe open a PR on the framework side cherry picking this commit https://github.com/kedro-org/kedro/pull/2485/commits/9c08611a781518f25fc78b8e9fe2ff6dc5c923b2